src/frontend/src/session.rs

```diff
@@ -359,6 +372,8 @@ impl SessionImpl {
         Self {
             env: FrontendEnv::mock(),
             database: "dev".to_string(),
+            user_name: "risingwave".to_string(),
```

Comment:
May use the global constants e.g. `DEFAULT_DATABASE_NAME` and `DEFAULT_SUPPER_USER`